__tests__/app.test.js

```diff
@@ -0,0 +1,25 @@
+const db = require("../db/connection")
+const seed = require("../db/seeds/seed")
+const testData = require("../db/data/test-data/index")
+const request = require("supertest")
+const app = require("../app")
+
+beforeEach(() => { return seed(testData)})
+afterAll(() => { return db.end() })
+
+describe("GET /api/topics", () => {
+    test.only("status: 200, responds with an array of topics with properties of slug and description", () => {
```

Comment:
Always remove - `.only` from tests - we will have to send them back otherwise as we need all tests to run to make sure your code works :)